### ISSUE TITLE
[Table chart] Mismatches between Library and GB block

### ISSUE THREAD

**Issue by rodica-andronache** [Codeinwp/visualizer#714]
These are the options missing from the Gutenberg block, for the Table chart.
- General Settings > Chart Description
- Table Settings > Enable paging display length
- Table Settings > Enable Search
- Table Settings > Enable Selection
- Table Settings > Enable Selection of
- Table Settings > Enable information of selection
- Row/Cell Settings > Header Row options
- Row/Cell Settings > Selected Table Item options

@contactashish13 regarding your comment here https://github.com/Codeinwp/visualizer/issues/619#issuecomment-641836196 is this the best way to make sure there are no more differences between the options provided in the library and in the Gutenberg block? I can test each type of chart and list the differences like in this case. Let me know

**Comment by contactashish13** [comment]
@rodica-andronache yes that would work unless you have a better method that you can suggest.
